assets/js/components/SettingsNotice/SettingsNoticeMultiRow.js

```diff
@@ -0,0 +1,57 @@
+/**
+ * Settings notice SettingsNoticeMultiRow component.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { getIconFromType } from './utils';
+
+const SettingsNoticeMultiRow = ( {
+	notice,
+	type,
+	Icon,
+	LearnMore,
+	children,
+} ) => {
+	return (
+		<div className="googlesitekit-settings-notice__row">
+			<div className="googlesitekit-settings-notice__icon">
+				{ Icon ? <Icon /> : getIconFromType( type ) }
+			</div>
+			<div>
+				<div className="googlesitekit-settings-notice__text">
+					{ notice }
+				</div>
+				<div className="googlesitekit-settings-notice__inner-row">
+					<div>
+						{ children }
+					</div>
+					{ LearnMore && (
+						<div className="googlesitekit-settings-notice__learn-more--align-bottom">
+							<div>
```

Comment:
Same here, let's remove this `div` if it is not needed.

Reply:
Nice! Neither was needed. I must have changed my styling as before they were needed :see_no_evil: 

I also just pushed some slight changes to the styling to fix some of the stories (Storybook should cover every example if you want to check :smile: )